CONTRIBUTING.md

```diff
@@ -0,0 +1,61 @@
+# Formspree Contribution Guide
+
+Thanks for your interest in contributing to Formspree! Please take a moment to review this document **before submitting a pull request.**
+
+If you want to contribute but aren't sure where to start, you can open a [new issue](https://github.com/formspree/formspree-react/issues).
+
+## Prerequisites
+
+This project uses [`yarn v1`](https://yarnpkg.com/) as a package manager.
+
+## Setting up your local repo
+
+Run the following commands from the root formspree-js directory:
+
+```sh
+yarn
+yarn build # generate the artifact needed for depedendent packages
+```
+
+## Development environment
+
+To play around with code while making changes, you can run the local development environment:
+
+```sh
+yarn dev
+```
+
+This will run an example app ([`examples/cra-demo`](../examples/cra-demo)) on [localhost:3000](http://localhost:3000) that uses create-react-app.
+
+## Development commands
+
+To run tests, typecheck on all packages:
+
+```sh
+yarn test
+yarn typecheck
+```
+
+## Opening a Pull Request
+
+When opening a pull request, include a changeset with your pull request:
+
+```sh
+yarn changeset
+```
+
+The changeset files will be committed to main with the pull request. They will later be used for releasing a new version of a package.
+
+_Note: non-packages (examples/\*) do not need changesets._
+
+## Releasing a new version
+
+_Note: Only core maintainers can release new versions of formpree-js packages._
+
+When we are ready to release a new version on one or more packages, follow these steps:
+
+1. Create a new branch off the `main` branch.
+1. Run `yarn changeset version` which will consume the changeset files and update the changelog and package.json files in the relevant packages.
```

Comment:
This is not quite accurate. Once a PR with a changeset is merged to `main`, our `release` GH workflow will create a "Version Packages" PR automatically ([example release run](https://github.com/formspree/formspree-js/actions/runs/5353146416/jobs/9708747974), [example PR](https://github.com/formspree/formspree-js/pull/40)).

This needs to be updated.

Reply:
Updated in c57537a